book/python/api.md

```diff
@@ -206,9 +206,9 @@ def partition(self, data):
 
 The TCP Sink Encoder is responsible for taking the output of the last computation in a pipeline and converting it into a `bytes` for Wallaroo to send out over a TCP connection.
 
-Your function must be decorated with `@encoder`.
+Your function must be decorated with `@wallaroo.encoder`.
 
-##### `encode(data)`
+##### `encoder(data)`
```

Comment:
This should stay as `encode` as it's a function, not a class. It should be named after the action it performs, not the object performing it

Reply:
Now that I think about it, this section is weird. It looks like it's left over from when `encode` was a field of the `Encoder` class, but isn't anymore. Maybe the entire section should be revised to reflect that it's just a decorated function?

Reply:
all of our examples use `encoder`, should we update all of those as well?